AppCreationScripts/apps.json

```diff
@@ -0,0 +1,36 @@
+{
+	"Sample": {
+		"Title": "Calling Microsoft Graph from a macOS app",
+		"Level": 400,
+		"Client": "macOS"
+	},
+	"AppRegistrations": [{
+		"x-ms-id": "active-directory-macOS-swift",
+		"x-ms-name": "macOS-swift",
+		"x-ms-version": "2.0",
+		"replyUrlsWithType": [{
+			"url": "msauth.<bundleId>://auth",
+			"type": "InstalledClient"
+		}],
+		"requiredResourceAccess": [{
+			"x-ms-resourceAppName": "Microsoft Graph",
+			"resourceAppId": "00000003-0000-0000-c000-000000000000",
+			"resourceAccess": [{
+				"id": "e1fe6dd8-ba31-4d61-89e7-88639da4683d",
+				"type": "Scope",
+				"x-ms-name": "user.read"
+			}]
+		}],
+		"codeConfigurations": [{
```

Comment:
does the bundleId also need to be replaced in Info.plist?

Reply:
Not for mac, but for iOS yes. I have replaced the bundle on iOS version

Reply:
it needs to happen on Mac as well since Redirect URI is based on bundle ID

Reply:
Let's chat tomorrow, just want to make sure I understand this part correctly

Reply:
As discussed, I have add the replacement for bundleId in app.json file